xml/System.IO/Directory.xml

```diff
@@ -550,8 +550,6 @@ An I/O error occurred.</exception>
   
  This method throws an <xref:System.IO.IOException> if the directory specified in the `path` parameter contains files or subdirectories.  
   
```

Comment:
```suggestion
  
The case-sensitivity of the `path` parameter corresponds to that of the file system on which the code is running. For example, it's case-insensitive on Windows and case-sensitive on Linux.
  
```

Reply:
If we refer to filesystem I feel like naming Windows and Linux is inprecise if not misleading.

Shouldn’t it mention NTFS and e.g. ext3?